DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: admiral
 Type: Package
 Title: ADaM in R Asset Library
-Version: 0.12.0
+Version: 0.12.0.9000
```

Comment:
@ddsjoberg this is failing the R version checks now, is there something we need to add to allow/turn on for the development versions?

Reply:
We have a non-standard R CMD Check that fails when the NOTE about dev versions appears. We'll need to update the R CMD Check workflow.